Localization/en-US.hjson

```diff
@@ -22,56 +16,86 @@ Mods: {
+			}
+		}
 
-		Config: {
-			$parentVal: Settings
-			ServerBlocked: Only the server owner can change the settings.
+		Configs: {
+			Config: {
+				DisplayName: Settings
+				ServerBlocked: Only the server owner can change the settings.
 
-			AddCheatSheetButton: {
-				$parentVal: Add Cheat Sheet/HERO's Mod button
-				Desc: Add a button that toggles difficulty mode to the Cheat Sheet mod and HERO's Mod.
-			}
+				CurrentFakedDifficulty: {
+					Label: Current Faked Difficulty
+					Tooltip: Makes the enabled settings think it is set to this difficulty.
+				}
 
-			ChangeAI: {
-				$parentVal: Change Boss AI
-				Desc:
-					'''
-					Bring Expert Mode boss AI into Normal Mode.
-					(Warning: may break some modded bosses)
-					'''
-			}
+				DropTreasureBagsInNormal: {
+					Label: Modify loot
+					Tooltip: Changes loot in accordence with your selected Beneficial Difficulty.
+				}
 
-			DropBags: Treasure Bags in Normal Mode
+				AddCheatSheetButton: {
+					Label: Add Cheat Sheet/HERO's Mod button
+					Tooltip: Add a button that toggles difficulty mode to the Cheat Sheet mod and HERO's Mod.
+				}
 
-			ExpertCommand: {
-				$parentVal: Add /difficulty command
-				Desc: Add the /difficulty chat command that toggles Expert Mode.
-			}
+				ChangeBossAI: {
+					Label: Change Boss AI
+					Tooltip:
+						'''
+						Bring Expert Mode boss AI into Normal Mode.
+						(Warning: may break some modded bosses)
+						'''
+				}
 
-			CurrentFakedDifficulty: {
-				$parentVal: Current Faked Difficulty
-				Desc: Makes the world think it is set to this difficulty.
-			}
+				SlotsWorksInNormal: {
+					Label: Accessory Slots
+					Tooltip: The extra accessory slot(s) work based on the current beneficial difficulty.
+				}
 
-			SlotsHack: {
-				$parentVal: Accessory Slots
-				Desc: The extra accessory slot(s) work in Normal Mode.
-			}
+				BossBlacklist: {
+					Label: Boss Blacklist
+					Tooltip: These NPCs won't be affected by Boss Expertise.
+				}
 
-			BossBlacklist: {
-				$parentVal: Boss Blacklist
-				Desc: These bosses won't be affected by Boss Expertise.
-			}
+				BossWhitelist: {
+					Label: Boss Whitelist
+					Tooltip:
+						'''
+						These NPCs will be affected by Boss Expertise
+						even if they're technically not bosses.
+						'''
+				}
+
+				FakedBeneficialDifficulty: {
+					Label: Faked Beneficial Difficulty
+					Tooltip: ""
+				}
+
+				ForTheWorthy: {
+					Label: For The Worthy
+					Tooltip: ""
+				}
+
+				ChangeNPCAI: {
+					Label: Change Enemy AI
+					Tooltip:
+						'''
+						Bring Expert Mode Enemy AI into Normal Mode.
+						(Warning: may break some modded bosses)
+						'''
+				}
 
-			BossWhitelist: {
-				$parentVal: Boss Whitelist
-				Desc:
-					'''
-					These NPCs will be affected by Boss Expertise
-					even if they're technically not bosses.
-					'''
+				AddExpertCommand: {
+					Label: Add /difficulty command
+					Tooltip: Add the /difficulty chat command that toggles Expert Mode.
```

Comment:
```suggestion
					Tooltip: Add the /difficulty chat command that changes the world's actual difficulty.
```

Reply:
the command changes the faked difficulty. I would need more reflection to modify the variable since last time I checked it is private/internal.

Reply:
Oh sorry I was mistaken. But I think originally that's basically what I wanted from the command. The fake difficulty being managed by the config and the actual difficulty being managed by the command.

Reply:
Btw there are a bunch of "resolved conversations" from my ru-RU lang commit, but there are also some unresolved convos that GitHub decided to hide for whatever reason.